benefits/enrollment/templates/enrollment/index.html

```diff
@@ -42,7 +42,7 @@ <h1 class="pb-lg-8 pb-4">
           .done(function() {
           $.get("{{ access_token_url }}", function(data) {
               $(".loading").remove();
-              $(".invisible").removeClass("invisible");
+              $(".invisible").removeClass("invisible").addClass("visible");
```

Comment:
Just curious, what was this change for?

Reply:
As I was building this page I kept seeing the result of this `.removeClass()` call and was super confused:

```html
<div class="">
...
</div>
```

I couldn't figure out where the div with the empty class was coming from! Once I realized it was this code, I thought it would be nice to add a class back to make it more explicit.

From https://getbootstrap.com/docs/5.1/utilities/visibility/

Reply:
I could have noted that better in the commit / comments though, I will add something as I'm fixing your other note.

Reply:
Split this change into its own commit for clarity: https://github.com/cal-itp/benefits/pull/1968/commits/1216af6c06a28bf403314fe291571a62b0d15653